csharp/Ice/Greeter/ServerAMD/Chatbot.cs

```diff
@@ -9,7 +9,7 @@ internal class Chatbot : GreeterDisp_
 {
     // Implements the abstract method greet from the GreeterDisp_ class generated by the Slice compiler.
```

Comment:
Should we use the actual name in the comment? there is no "greet" method in GreeDisp_

Reply:
Yes, will fix.